boot/zephyr/Kconfig

```diff
@@ -269,20 +261,13 @@ config BOOT_FIRMWARE_LOADER
 	  Note: The firmware loader image must be signed with the same signing
 	  key as the primary image.
 
-endchoice
-
-# Workaround for not being able to have commas in macro arguments
-DT_CHOSEN_Z_SRAM := zephyr,sram
-
-if BOOT_RAM_LOAD
-config BOOT_IMAGE_EXECUTABLE_RAM_START
-	hex "Boot image executable ram start"
-	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_SRAM))
+config BOOT_NO_UPGRADE
+	bool "No upgrade"
+	help
+	  No upgrade is performed - usually, this option will be used in
+	  conjunction with BOOT_RAM_LOAD.
```

Comment:
I don't know what you're doing here but no, this Kconfig is not needed and put the ram load Kconfig back where it was